src/js/stores/MesosStateStore.js

```diff
@@ -105,7 +105,7 @@ class MesosStateStore extends GetSetBaseStore {
   }
 
   getServiceFromName(name) {
-    const services = this.get("lastMesosState").frameworks;
+    const services = (this.get("lastMesosState") || {}).frameworks;
```

Comment:
what we could do instead is defining a getter `getLastMesosState` and have a default value there `{tasks, frameworks, slaves, executors}`. Then replacing all the usages `this.get("lastMesosState")` -> `this.getLastMesosState()`